src/chat/tools/getEnvironments.ts

```diff
@@ -25,6 +27,26 @@ export interface IGetEnvironmentsParameters {
 export class GetEnvironmentsTool extends BaseLanguageModelTool<IGetEnvironmentsParameters> {
   readonly name = "get_environments";
 
+  prepareInvocation(
+    options: LanguageModelToolInvocationPrepareOptions<IGetEnvironmentsParameters>,
+    token: CancellationToken,
+  ): ProviderResult<{ invocationMessage: string; confirmationMessage: string }> {
```

Comment:
Probably should declare `{ invocationMessage: string; confirmationMessage: string}` as a named type to reduce duplication later on down.

Reply:
Yep, should be able to use `PreparedToolInvocation` like in the `get_connections` tool: https://github.com/confluentinc/vscode/blob/50f1996a2e68714fd0cf5129d22b868c2cc28fe0/src/chat/tools/getConnections.ts#L47C21-L47C43